tests/template_generate_graphical_units_data.py

```diff
@@ -0,0 +1,11 @@
+from manim import *
+from tests.helpers.graphical_units import set_test_scene
+
+
+class YourClassHere(Scene):
+    def construct(self):
+        circle = Circle()
+        self.play(Animation(circle))
+
+
+set_test_scene(YourClassHere, "<whereitbelongs>")
```

Comment:
**Issue :** It should not be <whereitbelongs> but more something like `<module_name>` (see docstring for the function).

Reply:
solved